docs/contributing/contributing-releases/README.md

```diff
@@ -68,6 +68,8 @@ Follow the steps below to create an RC release.
 
 1. In the `radius-project/samples` repo, run the [Test Samples](https://github.com/radius-project/samples/actions/workflows/test.yaml) workflow. Run the workflow from the `edge` branch and using the Radius RC release version number being released.
 
+   > The `edge` branch needs to be upmerged with the previous release branch before this happens. To upmerge the `edge` branch, create a new branch off of the `edge` branch and run `git merge origin/v<previous-release-number>`. For example, if the current release is `v0.34`, the command would be `git merge origin/v0.33`. Then, push these changes to the new branch and create a PR against `edge`. Run the `Test Samples` workflow once the upmerge PR has been merged to `edge`. 
```

Comment:
This was done outside of release in the past, are we making it a part of the release process now? @sk593 @willtsai

Reply:
I'm not sure who is supposed to be in charge of doing the upmerge (if it's oncall or someone else). But it was an issue during the last release because the edge branch didn't have the latest changes (the samples workflow was failing because of this since the latest test fixes weren't in the edge branch). So I think we should at least document that it needs to be done before the samples workflow runs even if the actual process isn't part of the release

Reply:
The team synced offline, and it looks like this will be covered in the release process moving forward.

Reply:
Sounds good, thank you for following up on this!